sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -9,7 +9,6 @@
 )
 
 from azure.storage.blob._shared import sign_string, url_quote
-from azure.storage.blob._shared.constants import X_MS_VERSION
```

Comment:
I think we don't want to remove X_MS_VERSION, at least we should keep it as the default version

Reply:
but if you set x_ms_version=None, it will override the default one in the parent class SharedAccessSignature

Reply:
Yup, fixed.